src/app/components/search-form/search-form.component.html

```diff
@@ -11,6 +11,11 @@
         [matAutocomplete]="auto"
       />
       <mat-autocomplete class="light" #auto="matAutocomplete">
+        <p
+          class="autocomplete"
+          [style.marginLeft]="'16px'"
+        > <a routerLink="/busca-avancada"><b>Opções de Busca Avançada</b></a>
```

Comment:
Sugiro abrir em uma nova aba para não zerar outros valores possivelmente já preenchidos para "Município" e "Período".

---

[**English**]

I suggest opening in a new tab to avoid reseting other values possibly already filled in for "Municipality" and "Period".

```suggestion
        > <a routerLink="/busca-avancada" target="_blank"><b>Opções de Busca Avançada</b></a>
```